src/ui/home/components/collapsibleCredentialCard.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import { CardWrapper } from 'src/ui/structure'
+import { Spacing, Typography } from 'src/styles'
+import { DecoratedClaims } from 'src/reducers/account'
+import I18n from 'src/locales/i18n'
+import MoreIcon from 'src/resources/svg/MoreIcon'
+import { CredentialCard } from './credentialCard'
+import { credentialStyles } from './sharedConstants'
+
+interface Props {
+  onPress?: () => void
+  did: string
+  credential: DecoratedClaims
+  leftIcon: React.ReactNode
+}
+
+interface State {
+  collapsed: boolean
+}
+
+export class CollapsibleCredentialCard extends React.Component<Props, State> {
+  public constructor(props: Props) {
+    super(props)
+    this.state = {
+      collapsed: true,
+    }
+  }
+
+  private handleTouch = () => {
+    this.setState(prevState => ({ collapsed: !prevState.collapsed }))
+  }
+
+  public render() {
+    const { collapsed } = this.state
+    const { did, credential, onPress, leftIcon } = this.props
+
+    return (
+      <View onTouchEnd={this.handleTouch}>
```

Comment:
This could be simplified to `const CorrectCredentialCard = collapsed ? CollapsedCredentialCard : CredentialCard` since the props are all the same.

Reply:
Does that actually work? How would it look?